install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -43,3 +43,12 @@ spec:
             severity: warning
           annotations:
             message: "Kubelet health failure threshold reached"
+    - name: system-memory-exceeds-reservation
+      rules:
+        - alert: SystemMemoryExceedsReservation
+          expr: |
+            container_memory_rss{id="/system.slice"} > 1073741824
```

Comment:
Hrm, this needs to come from the reservation the system makes, which is exposed as a metric

Reply:
constant should be kube_node_status_capacity{... mem} - kube_node_status_allocatable{... mem}, and you need to figure out how to make that number show up in the annotation somehow

Reply:
is there anything on a node outside of system and kubepods?  should the first value be `container_memory_rss{id="/"} - container_memory_rss{id="/kubepods.slice"} > (capacity - allocatable)`

Reply:
all the kernel threads are in `/`. no userspace process though.